openshift-client/src/main/java/io/fabric8/openshift/client/OpenshiftAdapterSupport.java

```diff
@@ -54,63 +43,11 @@ public Boolean isAdaptable(Client client) {
    * @param config {@link OpenShiftConfig} OpenShift Configuration
    * @return         True if oapi is found in the root paths.
    */
-  static boolean isOpenShift(Client client, OpenShiftConfig config) {
-    return isOpenShiftAPIGroups(client, config);
+  public static boolean isOpenShift(OkHttpClient client, OpenShiftConfig config) {
+    return new BaseClient(client, config) {}.getApiGroups()
```

Comment:
One of the reasons I put it on http client handling on the config was for this call, so that it could use the BaseClient.  Now that the adapt logic is in the openshiftclient, does that mean we'll have to introduce a different mechanism here to adapt the http client for openshift use?

Reply:
Yes, I saw this after all of the changes.
We might need to find some other place for the adapter.